tests/e2e/tests/product-gallery/inner-blocks/product-gallery-pager/product-gallery-pager.block_theme.side_effects.spec.ts

```diff
@@ -0,0 +1,218 @@
+/**
+ * External dependencies
+ */
+import { test as base, expect } from '@woocommerce/e2e-playwright-utils';
+
+/**
+ * Internal dependencies
+ */
+
+import { ProductGalleryPage } from '../../product-gallery.page';
+
+const blockData = {
+	name: 'woocommerce/product-gallery-pager',
+	selectors: {
+		frontend: {
+			pagerBlock:
+				'div[data-block-name="woocommerce/product-gallery-pager"]',
+			pagerListContainer:
+				'ul.wp-block-woocommerce-product-gallery-pager__pager',
```

Comment:
`trunk` already contains the migrated the Pager classnames `wc-block-product-gallery-pager__pager` - let's make sure update them in the E2E tests too.

Reply:
Thank you for pointing this out, I just updated that 🙌